### PR TITLE
3.0.6 release

### PR DESCRIPTION
**Pull request by dingo-d**
This is a bugfix release that fixes the autowiring issue (exclude the non service classes).

Tested on a fresh project, no errors found.

### CODE REVIEW

**Review by iruzevic — CHANGES_REQUESTED** [review]
You are missing a change-log

**Review by IvanGrginovInf — DISMISSED** [review]

**Review by iruzevic — DISMISSED** [review]

**Comment by iruzevic** [comment]
After the merge please create a tag and update the release description in github with the content from the changelog

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-	"name": "@infinum/eightshift-libs",
-	"version": "3.0.5",
+	"name": "@infinum/eightshift-libs",g
```
Comment (iruzevic) [inline package.json:2]: a wild g appears :D

**Review by iruzevic — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dingo-d) [reply package.json]: Doh! Fixed

**Review by iruzevic — APPROVED** [review]

**Review by IvanGrginovInf — APPROVED** [review]

**Pull request merged**